src/project/internal/projectactionscontroller.cpp

```diff
@@ -378,7 +378,7 @@ bool ProjectActionsController::closeOpenedProject(bool quitApp)
 
 IInteractive::Button ProjectActionsController::askAboutSavingScore(INotationProjectPtr project)
 {
-    std::string title = qtrc("project", "Do you want to save changes to the score “%1” before closing?")
+    std::string title = qtrc("project", "Do you want to save changes to the score "%1" before closing?")
```

Comment:
Needs to be
std::string title = qtrc("project", "Do you want to save changes to the score \"%1\" before closing?")

Reply:
Done.